src/common/src/aws/sdk/kotlin/crt/io/TlsVersion.kt

```diff
@@ -1,17 +1,19 @@
 /*
  * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
- * SPDX-License-Identifier: Apache-2.0.
+ * SPDX-License-Identifier: Apache-2.0
  */
 
 package aws.sdk.kotlin.crt.io
 
+/* ktlint-disable enum-entry-name-case */
```

Comment:
**Comment**: Same comment as above—let's separate the v*n* part into a new "word" and avoid the rule overide.